serving/samples/helloworld-java/src/main/java/com/example/helloworld/HelloworldApplication.java

```diff
@@ -1,26 +1,13 @@
 package com.example.helloworld;
 
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import static spark.Spark.*;
 
-@SpringBootApplication
-public class HelloworldApplication {
-
-	@Value("${TARGET:World}")
-	String message;
-
-	@RestController
-	class HelloworldController {
-		@GetMapping("/")
-		String hello() {
-			return "Hello " + message + "!";
-		}
+public class HelloWorldApplication {
+	
+	public static void main(String args[]) {
+		port(8080);
```

Comment:
Convention for the helloworld samples:

* Respond to `/`
* Listen to the port specified by $PORT environment variable, with default to 8080
* Replace "World" with TARGET environment variable when present
* Be excited by ending the message with a bang!

Reply:
So SparkJava runs on default port [SparkJava port 4567](http://sparkjava.com/documentation#embedded-web-server)
So have to set the program to use 8080 instead of 4567.